### ISSUE TITLE
Blob query issue from Lua.

### ISSUE THREAD

**Issue by ImagicTheCat** [GHMatti/ghmattimysql#29]
**Describe the bug**
Blob are returned as a table of `string` => `number`. It took me a while to understand what was going on. I can work around that, but it can be confusing if other people are using Blob (maybe all binary) types.

**Expected behavior**
A list of bytes.

**Software:**
 - OS: Windows
 - Version of the resource: 1.2.3
 - Database Version: 10.1.21-MariaDB

**Comment by GHMatti** [comment]
How urgent is it for this fix to be pushed to release?

I want to do some more major refactoring of the code before the next release if possible.

Mainly due to a new feature: `store`. As for when you go for a lot players, export calls with more data, especially repetitive ones should be rather avoided.

See https://gist.github.com/GHMatti/9759400ebd6c85ddb5eefa215e5b70da for getting a gist why.

Basically you got about 5kB of data per server tick that can be transferred without the server starting to get issues due to messagepack. Hitches appear from 50kB+ per tick. A lot of repetitive query calls are avoidable just by storing an id for the query string and sending the parameters on call.

**Comment by ImagicTheCat** [comment]
Not urgent at all. I'm using this resource as a new DB driver for vRP, it's still experimental.
